hca/loader.py

```diff
@@ -10,28 +10,34 @@ def __init__(self, ingest: IngestApi):
 
     def get_project(self, project_uuid: str) -> HcaSubmission:
         hca_submission = HcaSubmission(HandleCollision.OVERWRITE)
+        self._map_project(hca_submission, project_uuid)
+        return hca_submission
+
+    def get_submission(self, submission_uuid: str) -> HcaSubmission:
+        hca_submission = HcaSubmission(HandleCollision.OVERWRITE)
+        self._map_submission(hca_submission, submission_uuid)
+        return hca_submission
+
+    def _map_project(self, hca_submission, project_uuid):
         project_type = 'projects'
-        project = self.__map_entity(hca_submission, project_type, project_uuid)
+        project = self._map_entity(hca_submission, project_type, project_uuid)
         submission_type = 'submissionEnvelopes'
         self.__map_link_type_to_link_names(hca_submission, project, submission_type)
         for submission_entity in hca_submission.get_entities(submission_type):
             self.__map_submission_manifests(hca_submission, submission_entity)
-        return hca_submission
 
-    def get_submission(self, submission_uuid: str) -> HcaSubmission:
-        hca_submission = HcaSubmission(HandleCollision.OVERWRITE)
+    def _map_submission(self, hca_submission, submission_uuid):
         submission_type = 'submissionEnvelopes'
-        submission_entity = self.__map_entity(hca_submission, submission_type, submission_uuid)
+        submission_entity = self._map_entity(hca_submission, submission_type, submission_uuid)
         self.__map_submission_manifests(hca_submission, submission_entity)
-        return hca_submission
 
     def __map_submission_manifests(self, hca_submission: HcaSubmission, submission_entity: Entity):
         manifest_type = 'bundleManifests'
         self.__map_link_type_to_link_names(hca_submission, submission_entity, manifest_type)
         for manifest_entity in hca_submission.get_entities(manifest_type):
             self.__map_manifest_content(hca_submission, manifest_entity)
 
-    def __map_entity(self, submission: HcaSubmission, entity_type: str, entity_uuid: str) -> Entity:
+    def _map_entity(self, submission: HcaSubmission, entity_type: str, entity_uuid: str) -> Entity:
```

Comment:
What is the difference between `_` and `__`?

Reply:
`__` private, not visible to instances or subclasses.
`_` not actually private but your IDE will raise a warning if you call directly.

Reply:
Thanks.